client/lib/abtest/active-tests.js

```diff
@@ -16,7 +16,7 @@ module.exports = {
 		allowExistingUsers: true,
 	},
 	coldStartReader: {
-		datestamp: '20160804',
+		datestamp: '20160824',
 		variations: {
 			noEmailColdStart: 20,
```

Comment:
Can we run this as a 33-33-34 test? 33% on noEmailColdStart, 33% on EmailColdStart, 34% on noChanges?

That way we can compare them against each other, and we'll get to significant results faster.